test/test.cpp

```diff
@@ -73,14 +73,14 @@ void clear_bucket(const ds3_client* client, const char* bucket_name) {
 
     for (i = 0; i < bucket_response->num_objects; i++) {
         if (bucket_response->objects[i].name->value[bucket_response->objects[i].name->size-1] == '/') {
-	    request = ds3_init_delete_folder(bucket_name, bucket_response->objects[i].name->value);
-	    error = ds3_delete_folder(client, request);
-	    ds3_free_request(request);
-	}else{
-	    request = ds3_init_delete_object(bucket_name, bucket_response->objects[i].name->value);
-	    error = ds3_delete_object(client, request);
-	    ds3_free_request(request);
-	}
+            request = ds3_init_delete_folder(bucket_name, bucket_response->objects[i].name->value);
+            error = ds3_delete_folder(client, request);
+            ds3_free_request(request);
+        }else{
```

Comment:
} else {

in general, its nice to see you fixing these formatting and style inconsistencies!